### DISCUSSION TITLE
How to add a global css variable?

### DISCUSSION THREAD

**Discussion by XiaoNode** [nuxt/nuxt#16213] [Questions]
How to add a global css variable, such as @gbobalColor: #dcdcdc in less, which can be used on all pages in Nuxt.js 3?

**Comment by IsraelOrtuno** [comment]
There are several ways to achieve this but you could just add a global variable to the `<style>` tag of your `App.vue` file:

```html
<style>
:root {
  --blue: #1e90ff;
  --white: #ffffff; 
}
</style>
```

Same if you add a css file to your app.

**Comment by sebalaini** [comment]
in my case for SCSS I use vite config

```
  vite: {
    css: {
      preprocessorOptions: {
        scss: {
          additionalData: '@use "@/assets/scss/setup.scss" as *;'
        }
      }
    }
```

where in that file I load all the SCSS mixin, Var etc...
